ex1/ex1-index.js

```diff
@@ -0,0 +1,40 @@
+// **Product rezolvă exercițiile într-un mediu de programare la alegere (ex. Replit, VS Code)**  **și încarcă codul pe Replit.**
+// **1. Scrieți o funcție care returnează pătratul unui număr**
+// Funcția `findSquare(num)` returnează pătratul numărului primit ca date de intrare (este transmis ca argument în apelul funcției).
+// Exemplu 1:
+// `Input: num = 6`
+// `Output: 36`
+// Exemplu 2:
+// `Input: num = 0`
+// `Output: 0`
+// Exemplu 3:
+// `Input: num = -12`
+// `Output: 144`
+//..................................................................................................................................
+
+console.clear()
+
+function theSquareOfInput1(number) { 
+
+       if (typeof number === 'number' && !isNaN(number)) {
+              return number = number ** 2;
+       }
+
+       return "Error: Input must be a number";
```

Comment:
```suggestion
       throw "Error: Input must be a number";
```

aici e mai binevenit `throw` decat `return`

Reply:
o sa tin cont ,4 spatii maxim la identare 🥂